controllers/comments-controllers.js

```diff
@@ -0,0 +1,23 @@
+//-------IMPORTS-------
+const {
+    fetchCommentsByArticleId
+}  = require('../models/comments-models');
+
+const {
+    checkArticleId
+} = require('../models/articles-models');
+
+//-----CONTROLLERS-----
+exports.getCommentsByArticleId = async (req, res, next) => {
+    const articleId = req.params.article_id;
+    try {
+        const { rows: article } = await checkArticleId(articleId);
+        if(article.length === 0) {
+            throw({ status: 404, msg: "Article ID not found"})

```

Comment:
I would always Promise.reject over throwing. It's more explicit that you're rejecting the promise chain